### PR TITLE
Create Remove-SPOTenantRenameSitePrioritization.md

### PR DESCRIPTION
**Pull request by AnfraMsft**
[PubOps PR review comments](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/614#issuecomment-2040116422)
***

Introduced documentation for the Remove cmdlet as part of site prioritization

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantRenameSitePrioritization.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ms.author: anfra
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Remove-SPOTenantRenameSitePrioritization
+## SYNOPSIS
+Allows you to remove the prioritization previously assigned to a site for early execution, as part of Advanced Tenant Rename.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantRenameSitePrioritization.md:17]: >Allows you to remove the prioritization [](http://example.com/codeflow?start=0&length=39)

Allows removal of the prioritization

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantRenameSitePrioritization.md:17]: >Advanced Tenant Rename [](http://example.com/codeflow?start=102&length=22)

do you want to inline href this like you did for Set-SPOTenantRenameSitePrioritization?

**Review by samkabue — APPROVED** [review]
:shipit:

**Comment by Stacyrch140** [comment]
Hi @AnfraMsft - 

**Following is a blocking issue that needs to be resolved before we can merge:**

sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOTenantRenameSitePrioritization.md
- Add this file to its corresponding TOC, and include the TOC with this PR. 

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=main)

**Blocking**
- When a new article is added, the same PR must also contain at least one updated TOC with an entry for the new article. Exceptions are allowed when the file has ROBOTS: NOINDEX, NOFOLLOW in the metadata.

***

After you resolve the blocking issues, remove the **Back to submitter** label and add the **Sign off** label again. 

If you need help resolving issues in this PR, post your question in the [Ask an Admin Teams Channel](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).

Thanks, 
Stacy

**Comment by American-Dipper** [comment]
@AnfraMsft  - The article still needs to be added to a table of contents, and that TOC file added to this PR, per https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/614#issuecomment-2040116422.

**Comment by maggierui** [comment]
> @AnfraMsft - The article still needs to be added to a table of contents, and that TOC file added to this PR, per [#614 (comment)](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/614#issuecomment-2040116422).

@AnfraMsft Hi @Anupam! I know it is confusing about the TOC because it took me a while to find the TOC file too: sharepoint-online.md  If you could add the following lines in `sharepoint-online.md` at its right location (alphabatized), we will be ready to merge:
`### [Remove-SPOTenantRenameSitePrioritization](Remove-SPOTenantRenameSitePrioritization.md)

{{Manually Enter Remove-SPOTenantCdnOrigin Description Here}}`


Thanks!

**Comment by American-Dipper** [comment]
> @American-Dipper It seems @AnfraMsft Anfram has already updated the TOC as shown in the commit history: Just to be clewar - this repo uses unconventional TOC files, which is `sharepoint-online.md` Is this the reason that you think the TOC was not updated? image
> 
> Can you please explain which TOC file you are looking for?

On a follow-up review, we check if the previous reviewer's comments were addressed. There was no response, so I assumed the TOC hadn't been updated. But I see that it has.

**Pull request merged**